(dp1
S'output'
p2
S"<type 'exceptions.NameError'> global name 'requests' is not defined"
p3
sS'layer'
p4
S'/home/volney/Projetos/web2py/web2py/applications/CifraDeCesar/controllers/default.py'
p5
sS'code'
p6
S'<code object <module> at 0x7f10747dc530, file "/home/volney/Projetos/web2py/web2py/applications/CifraDeCesar/controllers/default.py", line 10>'
p7
sS'snapshot'
p8
(dp9
sS'traceback'
p10
S'Traceback (most recent call last):\n  File "/home/volney/Projetos/web2py/web2py/gluon/restricted.py", line 219, in restricted\n    exec(ccode, environment)\n  File "/home/volney/Projetos/web2py/web2py/applications/CifraDeCesar/controllers/default.py", line 88, in <module>\n  File "/home/volney/Projetos/web2py/web2py/gluon/globals.py", line 414, in <lambda>\n    self._caller = lambda f: f()\n  File "/home/volney/Projetos/web2py/web2py/applications/CifraDeCesar/controllers/default.py", line 22, in index\n    session.message = encrypt(request.vars.chave, requests.vars.texto)\nNameError: global name \'requests\' is not defined\n'
p11
s.